filtersrv/src/main/java/org/apache/rocketmq/filtersrv/FiltersrvStartup.java

```diff
@@ -139,7 +139,7 @@ public static FiltersrvController createController(String[] args) {
 
             Runtime.getRuntime().addShutdownHook(new ShutdownHookThread(log, new Callable() {
                 @Override
-                public Object call() throws Exception {
+                public Void call() throws Exception {
                     controller.shutdown();
                     return null;
```

Comment:
When use the Void as the return signature, you don't need to use return in the function.  
Please remove the line of 144.